gradle-conjure/src/main/java/com/palantir/gradle/conjure/ConjureGeneratorTask.java

```diff
@@ -96,6 +97,7 @@ public void compileFiles() {
                 ImmutableList.Builder<String> commandArgsBuilder = ImmutableList.builder();
                 File thisOutputDirectory = outputDirectoryFor(file);
                 getProject().mkdir(thisOutputDirectory);
+                GFileUtils.cleanDirectory(thisOutputDirectory);
```

Comment:
Does this just wipe everything in the directory?

Reply:
Yes, its just a wrapper around `FileUtils.cleanDirectory` which recursively deletes everything in the directory